tools/monitoring/config/config.yaml

```diff
@@ -41,6 +41,20 @@ spec:
         prs-affected: 0
         period: 1440
 
+  - error-pattern: 'ERROR: timeout waiting for pods to come up'
+    hint: 'This may lead to Knative setup failure'
```

Comment:
For the hint, I think these are action items the oncall can take in order to fix the infrastructure issue. Could you please add more into how to fix this issue if you know potential issue/fix? 
This may be more of a case-by-case issue that the pods timeout. If you don't have enough info to come up with an oncall action item, then the current hint is good for now.

Reply:
I am not familiar with how to deal with pod not coming up. But adding the hint here helps build a connection with the Knative setup failure. So when the oncall sees the two failures together he got the hint of such a connection and investigate further. I think after an oncall investigated a problem he can give out a better recovery hint.